.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,86 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ['bug, untriaged']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Related component
+      description: Choose a specific OpenSearch component your bug belongs to.  If you are unsure which to select or if the component is not present, select "Other".
+      multiple: false
+      options:
+        - Other
+        - Build
+        - Clients
+        - Cluster Manager
+        - Extensions
+        - Indexing:Performance
+        - Indexing:Replication
+        - Indexing
+        - Libraries
+        - Plugins
+        - Search:Aggregations
```

Comment:
Could we simplify these to a much smaller list? I think for routing purposes we want each of these options to map to 1 set of owners - who could add the Search: Performance tag when they mark it triaged.

With a larger list it would be easy to constantly adding/removing items from this list based on priorities of the moment.

Reply:
My understanding is that each of these sub categories will have a separate owner no? If this is the case why not skip the extra triaging? Indexing/search could still be selected if the sub is not known.

Reply:
Yes, I prefer to keep the breakdown for better tracking and ownership. For example: Search: aggregation is totally different area than Search:Remote Search, each one has different code components and different subject matter expert.

Reply:
This list is not for people who know what the different areas are, its for those who _don't_ know; if this list reads like Amazon OpenSearch's organization that sounds like a bug not a feature.

Reply:
Folks opening issues will have a high level idea on which components it will map to (relevance, indexing, search etc.). For those who are unable to attach a label, we can have the maintainers help with triage.

Reply:
If we have the extended set of labels as options in the repo I think it should be included here.  I understand the ask to keep it simple, but I still think if we can reduce some manual routing effort we should try it.

Lets keep it as is to start? We can always change if we see it not working - ie. if too many issues are left as "Other"